data/data/aws/vpc/sg-master.tf

```diff
@@ -1,5 +1,9 @@
+locals {
+  description = "Created By OpenShift Installer"
```

Comment:
Move this to `data/data/aws/vpc/common.tf` rather than repeating it in sg-master.tf and sg-worker.tf. Local values are visible to everything in the module and not just to the file in which they are declared.

Do the same in `data/data/gcp/network` and `data/data/openstack/topology`.